presto-main/src/main/java/com/facebook/presto/sql/planner/RelationPlanner.java

```diff
@@ -777,7 +775,7 @@ public SetOperationPlanner(SetOperation node, Void context)
             return outputSampleWeight;
         }
 
-        public SetOperationPlanner process()
+        public SetOperationPlanner process(Void context)
```

Comment:
do you need any parameter at all? You can just pass `null` to `processAndCoerceIfNecessary`